test/helpers/cons.go

```diff
@@ -255,15 +255,7 @@ var ciliumKubCLICommands = map[string]string{
 	"cilium kvstore get cilium --recursive": "kvstore_get.txt",
 }
 
-// etcdDeploymentFiles are the list of files used to deploy etcd operator.
-var etcdDeploymentFiles = []string{
-	"00-crd-etcd.yaml",
-	"cilium-etcd-sa.yaml",
-	"cluster-role-binding-template.yaml",
-	"cluster-role-template.yaml",
-	"deployment.yaml",
-	"cilium-etcd-cluster.yaml",
-}
+const ciliumEtcdOperator = "https://raw.githubusercontent.com/cilium/cilium-etcd-operator/1.2/cilium-etcd-operator.yaml"
```

Comment:
I'm against of using a file from the web, the reasons:

- In case of fails, what does it fail, the internet connectivity?
- In case of special needs for different kuberentes version, how we're going to handle that. 
- What happens if a backport breaks the 1.2 branch, all the test will fail. 
- If I need to submit a PR to fix something related to that, how can I do that?

On the other hand, I would not use a simple file meanwhile we use the `kubectl patch`, the main reason, you can't patch multiple documents with kubectl patch. 

At the moment we use some files from internet, but for me is a bad idea, getting things from internet can break the CI in the future. IMHO

Regards